platform/android/MapLibreAndroid/src/main/java/org/maplibre/android/maps/NativeMapView.java

```diff
@@ -44,7 +44,7 @@
 import java.util.List;
 
 // Class that wraps the native methods for convenience
-final class NativeMapView implements NativeMap {
+public final class NativeMapView implements NativeMap {
```

Comment:
Should we drop the `final` so it's easier to create a custom NativeMap with inheritance?

Reply:
Let me do some testing, I'm currently uncertain if the JNI bindings will work as expected with a derived class.